src/main/java/by/custom/utilcalculator/controller/MessageRouter.java

```diff
@@ -33,32 +35,33 @@ public SendMessage route(Update update) {
+    private SendMessage getCheckInputMessageAndGetAnswer(final Message message) {
         String usersMessage = message.getText();
-        String userID = message.getChatId().toString();
-        UserProgress userProgress = UserStorageManager.getInstance().getUserProgress(userID);
+        String chatID = message.getChatId().toString();
         String answer;
-
-        switch (usersMessage) {
-            case Command.START -> answer = getGreetingMessage();
-            case Command.EAES, Command.OTHER_COUNTRIES ->
-                    answer = botFieldsManager.processCarOrigin(usersMessage, userProgress);
-            case Command.PHYSICAL_PERSON, Command.JURIDICAL_PERSON ->
-                    answer = botFieldsManager.processOwnerType(usersMessage, userProgress);
-            case Command.LESS_3_YEARS_AGE, Command.BETWEEN_3_AND_7_YEARS_AGE, Command.MORE_7_YEARS_AGE ->
-                    answer = botFieldsManager.processCarAge(usersMessage, userProgress);
-            case Command.GASOLINE_TYPE_ENGINE, Command.ELECTRIC_TYPE_ENGINE ->
-                    answer = botFieldsManager.processEngineType(usersMessage, userProgress);
-            case Command.VOLUME_LESS_1000_CM, Command.VOLUME_BETWEEN_1000_2000_CM, Command.VOLUME_BETWEEN_2000_3000_CM,
-                    Command.VOLUME_BETWEEN_3000_3500_CM, Command.VOLUME_MORE_3500_CM ->
-                    answer = botFieldsManager.processEngineVolume(usersMessage, userProgress);
-            default -> answer = getSorryMessage();
+        try {
+            switch (usersMessage) {
+                case Command.START -> {
+                    userProgressManager.createNewUserProgress(chatID);
+                    answer = getGreetingMessage();
+                }
+                case Command.EAES, Command.OTHER_COUNTRIES ->
+                        answer = userProgressManager.processCarOrigin(usersMessage, chatID);
+                case Command.PHYSICAL_PERSON, Command.JURIDICAL_PERSON ->
+                        answer = userProgressManager.processOwnerType(usersMessage, chatID);
+                case Command.LESS_3_YEARS_AGE, Command.BETWEEN_3_AND_7_YEARS_AGE, Command.MORE_7_YEARS_AGE ->
+                        answer = userProgressManager.processCarAge(usersMessage, chatID);
+                case Command.GASOLINE_TYPE_ENGINE, Command.ELECTRIC_TYPE_ENGINE ->
+                        answer = userProgressManager.processEngineType(usersMessage, chatID);
+                case Command.VOLUME_LESS_1000_CM, Command.VOLUME_BETWEEN_1000_2000_CM, Command.VOLUME_BETWEEN_2000_3000_CM,
+                        Command.VOLUME_BETWEEN_3000_3500_CM, Command.VOLUME_MORE_3500_CM ->
+                        answer = userProgressManager.processEngineVolume(usersMessage, chatID);
+                default -> answer = getSorryMessage();
+            }
+        } catch (UserFileNotFoundException e) {
+            return new CustomSendMessage(chatID, e.getMessage());
```

Comment:
Thibk about to translate it here. Exception contains only error code.

Reply:
Error code was added